source/javascripts/app/_search.js

```diff
@@ -6,6 +6,8 @@
 
   var content, searchResults;
   var highlightOpts = { element: 'span', className: 'search-highlight' };
+  var search_delay = 0;
+  var timeout_handle = 0;
```

Comment:
Maybe worth using `camelCase` instead of `snake_case` on these variable names, just because the rest of the file does that? For `determine_search_delay` and `search_input` below, too.